index.bs

```diff
@@ -3134,6 +3217,66 @@ and an optional [=boolean=] |isNoised| (default false):
 1. Run [=obtain and deliver an aggregatable debug report on source registration=]
     with |dataTypes|, |source|, and |isNoised|.
 
+To <dfn>find sources with common destinations and reporting origin</dfn> given an [=attribution source=] |pendingSource|:
+1. Let |matchingSources| be a new [=list=].
+1. [=set/iterate|For each=] |source| of the user agent's [=attribution source cache=]:
+    1. Let |commonDestinations| be the [=set/intersection=] of |source|'s [=attribution source/attribution destinations=] and |pendingSource|'s [=attribution source/attribution destinations=].
+    1. If |commonDestinations| [=set/is empty=], [=iteration/continue=].
+    1. If |source|'s [=attribution source/reporting origin=] and |pendingSource|'s [=attribution source/reporting origin=] are not [=same origin=], [=iteration/continue=].
+    1. [=list/Append=] |source| to |matchingSources|.
+1. Return |matchingSources|.
+
+To <dfn>check if source has incompatible attribution scope fields</dfn> given an [=attribution source=] |source| and an [=attribution source=] |pendingSource|:
+1. If |pendingSource|'s [=attribution source/attribution scope limit=] is null, return false.
```

Comment:
Shall we assert that |source|'s attribution scope limit is null under this condition?

Reply:
I don't think so. the case you mentioned should already be handled in line 3723 - 3725. For this scenario, we are not deactivating the sources for the scenario, but removing the attribution scopes for |source|.

Reply:
Done.